hassio/src/addon-store/hassio-addon-store.ts

```diff
@@ -104,12 +107,8 @@ class HassioAddonStore extends LitElement {
     }
   }
 
-  static get styles(): CSSResult {
```

Comment:
Why remove this?

Reply:
From what I saw when testing, this margin did nothing. Not sure if it was a coincidence or not. I'll look into it

Reply:
Ended up reverting in the end

Reply:
Not sure it did anything, but I've added it back in to not cause any adverse effects